sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisSuite.scala

```diff
@@ -217,5 +217,23 @@ class AnalysisSuite extends AnalysisTest {
       nullResult,
       udf4)
     // checkUDF(udf4, expected4)
+    }
+
+  test("SPARK-11863 mixture of aliases and real columns in orderby clause - tpcds 19,55,71") {
+    val a = testRelation2.output(0)
+    val c = testRelation2.output(2)
+    val alias1 = a.as("a1")
+    val alias2 = c.as("a2")
+    val alias3 = count(a).as("a3")
+
+    val plan = testRelation2.
+      groupBy('a, 'c) ('a.as("a1"), 'c.as("a2"), count('a).as("a3")).
+      orderBy('a1.asc, 'c.asc)
```

Comment:
code style: we should put the `.` at the beginning of a line, not at the end. And also remove the space between `groupBy('a, 'c)` and `('a.as("a1"),...`